packages/amplify-category-geo/resources/custom-geofence-collection-resource-handler.js

```diff
@@ -1,18 +1,17 @@
 const response = require('cfn-response');
 const aws = require('aws-sdk');
-
 exports.handler = async function (event, context) {
   try {
-    console.log(`REQUEST RECEIVED:${JSON.stringify(event)}`);
+    console.log('REQUEST RECEIVED:' + JSON.stringify(event));
```

Comment:
Is this file supposed to be updated?

Reply:
hm no
I just undo-ed the linter changes on this file from the other PR as they may show up that we modified this file when we merge to dev branch.

Reply:
I think we should keep the lint fixes. It's okay if they bloat the final PR, it's why we're reviewing each PR as it gets merged to the feature branch.